flink/v1.16/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -613,8 +617,9 @@ static CatalogTable toCatalogTable(Table table) {
     // may use
     // CatalogTableImpl to copy a new catalog table.
     // Let's re-loading table from Iceberg catalog when creating source/sink operators.
-    // Iceberg does not have Table comment, so pass a null (Default comment value in Flink).
-    return new CatalogTableImpl(schema, partitionKeys, table.properties(), null);
+    Map<String, String> prop = Maps.newHashMap(table.properties());
+    String comment = prop.remove(COMMENT);
```

Comment:
Iceberg does not have Table comment, and we never seem to get a value here. We don't seem to need to change this.